src/ds/d400/d400-options.cpp

```diff
@@ -67,8 +67,16 @@ namespace librealsense
         return option_range{ -40, 125, 0, 0 };
     }
 
-    auto_exposure_limit_option::auto_exposure_limit_option(hw_monitor& hwm, sensor_base* ep, option_range range, std::shared_ptr<limits_option> exposure_limit_enable)
-        : option_base(range), _hwm(hwm), _sensor(ep), _exposure_limit_toggle(exposure_limit_enable)
+    auto_exposure_limit_option::auto_exposure_limit_option( hw_monitor & hwm,
```

Comment:
Can we place in the feature file?
Or does it really complicate things?

Reply:
We can place non generic options in the feature file, but we need to consider the right place for the feature as well.
This is a d400 option, is the feature d400 only? Is it relevant to d500? If not then `ds/features` is not the right place for it.